bin/extract_ietf_modules.py

```diff
@@ -151,29 +151,29 @@ def main():
         remove_directory_content(dir, debug_level)
 
     # Extract YANG models from IETF RFCs files
-    rfcExtractor = RFCExtractor(rfc_path, args.rfcyangpath, args.rfcextractionyangpath, debug_level)
-    rfcExtractor.extract()
-    rfcExtractor.clean_old_RFC_YANG_modules(args.rfcyangpath, args.yangexampleoldrfcpath)
+    rfc_extractor = RFCExtractor(rfc_path, args.rfcyangpath, args.rfcextractionyangpath, debug_level)
+    rfc_extractor.extract()
+    rfc_extractor.clean_old_RFC_YANG_modules(args.rfcyangpath, args.yangexampleoldrfcpath)
     custom_print('Old examples YANG modules moved')
     custom_print('All IETF RFCs pre-processed')
 
     # Extract YANG models from IETF draft files
-    draftExtractor = DraftExtractor(draft_extractor_paths, debug_level)
-    draftExtractor.extract()
-    draftExtractor.dump_incorrect_drafts(public_directory)
+    draft_extractor = DraftExtractor(draft_extractor_paths, debug_level)
+    draft_extractor.extract()
+    draft_extractor.dump_incorrect_drafts(public_directory)
```

Comment:
What about loading value of the second parameter - `send_emails_about_problematic_drafts` from config file.
To the future for example if there will be some problem with sending emails, we can simply change this value in config file from True to False, without need of rebuilding code.